<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="LAPTOP-0M17UIVA" failures="1" tests="10" name="failtestscripts.FailScriptsExecution" time="0.022" errors="0" timestamp="2023-12-06T20:28:53 IST" skipped="5">
  <testcase classname="failtestscripts.FailScriptsExecution" name="verifyTestCase_4" time="0.000"/>
  <testcase classname="failtestscripts.FailScriptsExecution" name="verifyTestCase_1" time="0.007"/>
  <testcase classname="failtestscripts.FailScriptsExecution" name="verifyTestCase_2" time="0.002"/>
  <testcase classname="failtestscripts.FailScriptsExecution" name="verifyTestCase_5" time="0.001"/>
  <testcase classname="failtestscripts.FailScriptsExecution" name="verifyTestCase_3" time="0.000">
    <skipped/>
  </testcase> <!-- verifyTestCase_3 -->
  <testcase classname="failtestscripts.FailScriptsExecution" name="verifyTestCase_3" time="0.000">
    <skipped/>
  </testcase> <!-- verifyTestCase_3 -->
  <testcase classname="failtestscripts.FailScriptsExecution" name="verifyTestCase_3" time="0.001">
    <skipped/>
  </testcase> <!-- verifyTestCase_3 -->
  <testcase classname="failtestscripts.FailScriptsExecution" name="verifyTestCase_3" time="0.000">
    <failure message="expected [true] but found [false]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertTrue(Assert.java:57)
at org.testng.Assert.assertTrue(Assert.java:67)
at failtestscripts.FailScriptsExecution.verifyTestCase_3(FailScriptsExecution.java:21)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
at org.testng.internal.invokers.TestInvoker.retryFailed(TestInvoker.java:248)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:62)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:957)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:200)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
at org.testng.TestRunner.privateRun(TestRunner.java:848)
at org.testng.TestRunner.run(TestRunner.java:621)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:443)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:437)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:397)
at org.testng.SuiteRunner.run(SuiteRunner.java:336)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1280)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1200)
at org.testng.TestNG.runSuites(TestNG.java:1114)
at org.testng.TestNG.run(TestNG.java:1082)
at com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:66)
at com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:105)
]]>
    </failure>
  </testcase> <!-- verifyTestCase_3 -->
  <testcase classname="failtestscripts.FailScriptsExecution" name="verifyTestCase_3" time="0.011">
    <skipped/>
  </testcase> <!-- verifyTestCase_3 -->
  <testcase classname="failtestscripts.FailScriptsExecution" name="verifyTestCase_3" time="0.000">
    <skipped/>
  </testcase> <!-- verifyTestCase_3 -->
</testsuite> <!-- failtestscripts.FailScriptsExecution -->
